rbac/management/querysets.py

```diff
@@ -63,7 +63,7 @@ def get_group_queryset(request):
         if username != identity_username:
             return Group.objects.none()
         else:
-            return Group.objects.filter(principals__username=username)
+            return Group.objects.filter(principals__username=username) | Group.platform_default_set()
```

Comment:
Just to capture the discussion from the review meeting, I think we can remove this in favor of just having the union with Group.platform_default_set() in group/view.py

Reply:
Thanks for the reminder/review! I'll push an update.

Reply:
@wcmitchell updated!